retail/retail-java-applications/terraform/README.MD

```diff
@@ -0,0 +1,116 @@
+# Deploy using Terraform
+
+Use Terraform to deploy the folllowing services defined in the `main.tf` file
+
+- Pub/Sub
+
+## Install Terraform
+
+If you are using the Google Cloud Shell Terraform is already installed.
```

Comment:
Link to Cloud Shell documentation please.

Reply:
Added a link to cloud shell documentation.